src/components/ScannerExamples.js

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+import { useStaticQuery, graphql } from "gatsby"
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import 'react-tabs/style/react-tabs.css';
+
+const PREFIX = 23
```

Comment:
Can you replace this static 23 with the string and a .length call. That would make it clearer what this prefix acutally is.

Example:
```js
 const PREFIX = 'foobar'.length
```